jbpm-container-test/jbpm-in-container-test/jbpm-container-integration-deps/rest/pom.xml

```diff
@@ -25,6 +25,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>jakarta.ws.rs</groupId>
+      <artifactId>jakarta.ws.rs-api</artifactId>
```

Comment:
Should we limit the scope to test?

Reply:
It is a dependency for the archive being deployed to the container. Otherwise it wouldn't be included in the final archive at all.

Reply:
OK, clarified:
It is a dependency for the archive being deployed to the container. Otherwise it wouldn't be included at all.
shrinkwrap resolves dependencies based on this profile and then adds them to the archive

Reply:
Thanks for clarification